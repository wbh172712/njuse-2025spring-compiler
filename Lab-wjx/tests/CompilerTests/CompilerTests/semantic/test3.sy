int a = 1;
int b;
int c[2] = {1, 2};

int main() {
  int a = 1;
  if (a) {
    int b = a;
  }
}