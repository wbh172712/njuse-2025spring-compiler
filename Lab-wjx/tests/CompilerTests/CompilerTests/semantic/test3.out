No semantic errors in the program!
